types/setting.go

```diff
@@ -1064,8 +1064,7 @@ var (
 			"List the enabled resource types in a semicolon-separated list. \n\n" +
 			"Available items are: \n\n" +
 			"- **replicaData**: replica data store \n\n" +
```

Comment:
BTW, IIRC Longhorn doesn't use camel casing for the setting values, we should fix the `replicaData` for consistency.

Reply:
which one is better "replica-data", "replica-data" or "replica data"?

Reply:
Thanks. Will change to `replica-data`.